sharktank/sharktank/models/clip/export.py

```diff
@@ -67,7 +72,7 @@ def export_clip_text_model_dataset_from_hugging_face(
 
 
 def export_clip_text_model_mlir(
-    model: Union[ClipTextModel, PathLike],
+    model: Union[ClipTextModel, AnyPath],
```

Comment:
Leave this as `PathLike` and remove the `AnyPath`. Users should just make sure to convert the string before invoking. (Same as above)

Reply:
I removed AnyPath.